go.mod

```diff
@@ -4,6 +4,6 @@ go 1.13
 
 require (
 	github.com/newrelic/newrelic-telemetry-sdk-go v0.2.0
-	go.opentelemetry.io/otel v0.4.3
+	go.opentelemetry.io/otel v0.5.0
```

Comment:
Any chance we can bump this to `v0.6.0`? That is the latest from OTel and I think the changes you include here are all that are needed for that version as well.

Reply:
Ah, I forgot about the switch to new metric instruments in `v0.6.0`. That switch should be tackled in another PR.

Reply:
Sounds good.